src/redist/targets/BundledTemplates.targets

```diff
@@ -20,6 +20,21 @@
 
   </Target>
 
+  <ItemGroup>
+    <Bundled80Templates Include="Microsoft.DotNet.Common.ItemTemplates" PackageVersion="$(MicrosoftDotNetCommonItemTemplates80PackageVersion)" />
```

Comment:
`MicrosoftDotNetCommonItemTemplates80PackageVersion` is `7.0.100-rc.1.22425.9` and this package exists. Why it is not included? 

Most of the failures are about `tool-manifest` template which is part of this package. If installer is fixed to include it, which seems to be not happening now, the test failures may be gone.

Reply:
Thanks @vlada-shubina for your help here. I think before it might be the migration was trying `ItemTemplates8.0` instead of a generic `ItemTemplates`? I don't understand why `MicrosoftDotNetCommonItemTemplates80PackageVersion` would point to a `7.1` version, but sounds good for now. 

Do you know if the semantics are the same for `Microsoft.DotNet.Test.ProjectTemplates.8.0`, so instead it should be authored as `Microsoft.DotNet.Test.ProjectTemplates`?

Reply:
For item templates, only single package exists `Microsoft.DotNet.Common.ItemTemplates`.
For project templates, we do have package for each framework: `Microsoft.DotNet.Common.ProjectTemplates.8.0`, so for installer for .NET 8 this should be included, however it is not ready it, so for time being I suggest to include 7.0 instead (`Microsoft.DotNet.Common.ProjectTemplates.7.0`). Once https://github.com/dotnet/sdk/pull/27486 is unblocked, I can change it here to include correct package,

Reply:
Test templates have separate package for each version.

Reply:
Also discussed with @nohwnd: the preference for test templates is to have them excluded until 8.0 version is ready (should be available very soon).